src/gcode-machine-control.cc

```diff
@@ -530,13 +529,21 @@ void GCodeMachineControl::Impl::set_output_flags(HardwareMapping::LogicOutput ou
 }
 
 void GCodeMachineControl::Impl::get_current_position() {
-  if (planning_buffer_.size() > 0) {
-    const struct AxisTarget *current = planning_buffer_[0];
-    const int *mpos = current->position_steps;
-    const float x = 1.0f * mpos[AXIS_X] / cfg_.steps_per_mm[AXIS_X];
-    const float y = 1.0f * mpos[AXIS_Y] / cfg_.steps_per_mm[AXIS_Y];
-    const float z = 1.0f * mpos[AXIS_Z] / cfg_.steps_per_mm[AXIS_Z];
-    const float e = 1.0f * mpos[AXIS_E] / cfg_.steps_per_mm[AXIS_E];
+    int i, motor;
+    const MotorCoordSystem
```

Comment:
I think it is easier to read if you break things like this
   const Type variable
           =  things_to_assign();